packages/kolibri-common/components/Cards/AccessibleFolderCard.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+
+  <KCard
+    :to="to"
+    :headingLevel="headingLevel"
+    :layout="computedLayout"
+    thumbnailDisplay="large"
+    :title="contentNode.title"
+    :thumbnailSrc="thumbnailSrc"
+    thumbnailScaleType="centerInside"
+    thumbnailAlign="right"
+    :style="{ margin: '16px 0 16px 0', maxWidth: '800px' }"
+  >
+    <template #thumbnailPlaceholder>
+      <div class="default-folder-icon">
+        <KIcon
+          icon="topic"
+          :color="$themePalette.grey.v_600"
+        />
+      </div>
+    </template>
+
+    <template #belowTitle>
+      <div
+        class="header-bar"
+        :style="headerStyles"
+      >
+        <KIcon
+          icon="topic"
+          :color="$themePalette.grey.v_800"
+          class="folder-header-bar"
+        />
+        <p
+          class="folder-header-text"
+          :style="{ color: $themePalette.grey.v_600 }"
+        >
+          {{ coreString('folder') }}
+        </p>
+      </div>
+    </template>
+  </KCard>
+
+</template>
+
+
+<script>
+
+  import { validateLinkObject } from 'kolibri.utils.validators';
+  import useKResponsiveWindow from 'kolibri-design-system/lib/composables/useKResponsiveWindow';
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+
+  export default {
+    name: 'AccessibleFolderCard',
+    mixins: [commonCoreStrings],
+    setup() {
+      const { windowBreakpoint } = useKResponsiveWindow();
+
+      return {
+        windowBreakpoint,
+      };
+    },
+    props: {
+      to: {
+        type: Object,
+        required: true,
+        validator: validateLinkObject,
+      },
+      contentNode: {
+        type: Object,
+        required: true,
+      },
+      headingLevel: {
+        type: Number,
+        required: true,
+      },
+      thumbnailSrc: {
+        type: String,
+        default: null,
+      },
+    },
+    computed: {
+      headerStyles() {
+        return {
+          color: this.$themeTokens.text,
+          borderRadius: '4px',
+          height: '24px',
+          width: '74px',
+          margin: '0',
+          backgroundColor: this.$themePalette.grey.v_50,
+        };
+      },
+      computedLayout() {
+        if (this.windowBreakpoint === 0) {
```

Comment:
@AllanOXDi There's basically the same condition on line 93 as on line 97.

As @marcellamaki demonstrated in the code sample earlier, this will be done rather outside the card on grid level later on (these cards will be used in different contexts), so let's just remove `computedLayout` before we merge. We can abstract some of the behavior inside the cards themselves if there's some strong pattern later on.